src/pymmcore_plus/mda/_engine.py

```diff
@@ -32,6 +39,9 @@ def setup_sequence(self, sequence: MDASequence) -> None:
 
         self._mmc = self._mmc or CMMCorePlus.instance()
 
+        # switch off autofocus device if it is on
+        self._mmc.enableContinuousFocus(False)
```

Comment:
is there ever a case where we wouldn't want to turn this off?  Like, what about a multi-channel timelapse at a single XYZ position.  If the user had it on to begin with, shouldn't we just leave it on?

Reply:
I did some tests with the Nikon PFS and if the PFS is on before running     `self._mmc.setPosition(action.autofocus_device_name, action.autofocus_motor_offset)`, it does not precisely go to the correct position. This is why I added this line.

Reply:
right, so, what I'm saying is that it clearly does make sense in cases where you _know_ you are going to use autofocus at some point.  But here you've disabled autofocus *all* the time, regardless of anything.  So, if a user had manually turned on autofocus, and hadn't asked for any autofocus in their MDASequence, wasn't moving positions or anything, this would still turn it off on them.  That doesn't seem right?

Reply:
oh that's true! Ok so we can move this inside the `exec_event` if the event is an autofocus event I guess!

Reply:
it's certainly fine to have it here too, but you need to guard with some kind of conditional at least

Reply:
I want to do few test on the microscope...